test/common/testlib.py

```diff
@@ -732,7 +745,10 @@ def snapshot(self, title, label=None):
             self.cdp.command("clearExceptions()")
 
             filename = "{0}-{1}.png".format(label or self.label, title)
-            ret = self.cdp.invoke("Page.captureScreenshot", no_trace=True)
+            if self.body_clip:
+                ret = self.cdp.invoke("Page.captureScreenshot", clip=self.body_clip, no_trace=True)
```

Comment:
Why is that important? This just takes screenshots for human consumption, these are not involved in pixel tests.

Reply:
It's not important. Without this, the snapshots would contain some white margin because the window is bigger than the top-level `body`.